Packs/UBIRCH/IncidentTypes/incidenttype-Authenticity.json

```
@@ -0,0 +1,23 @@
+{
+    "id": "Authenticity",
+    "version": -1,
+    "vcShouldIgnore": false,
+    "locked": false,
+    "name": "Authenticity",
+    "prevName": "Authenticity",
```

```suggestion
    "id": "UBIRCH Authenticity",
    "version": -1,
    "vcShouldIgnore": false,
    "locked": false,
    "name": "UBIRCH Authenticity",
    "prevName": "UBIRCH Authenticity",
```

Please add a prefix to all custom incident fields and types.

---

@teizenman done